tox.ini

```diff
@@ -7,3 +7,4 @@ commands=
 deps=
     pytest
     pytest-cov
+	wrapt
```

Comment:
thanks, it was a tab, replaced with spaces, probably need to set it up in my editor